step.yml

```diff
@@ -198,6 +198,16 @@ inputs:
     - "no"
     is_required: true
 
+- manage_version_and_build_number: "no"
+  opts:
+    category: IPA export configuration
+    title: Manage version and build number
+    summary: This will change the version and build number of all content in your app. If your app has an invalid number (like one that was used previously, or precedes your current build number), the step provides an option to automatically increment it to a valid number. The step does not work if export options plist content has been set.
```

Comment:
It's not clear to me what the "yes" and "no" options mean based on this description. What does "manage" mean exactly, does Xcode increment the build number automatically?

Reply:
Updated the description. let me know what you think